test/functional/feature_rbf.py

```diff
@@ -29,15 +29,13 @@ def set_test_params(self):
         # both nodes disable full-rbf to test BIP125 signaling
```

Comment:
This comment is outdated

Reply:
oops, fixed and removed more vestigial signaling code and comments in test